src/native/http_request_utils.h

```diff
@@ -36,6 +36,8 @@ struct aws_http_message *aws_http_request_new_from_java_http_request(
     jobjectArray jni_headers,
     jobject jni_body_stream);
 
+jobject aws_java_http_request_from_native(JNIEnv *env, struct aws_http_message *message);
```

Comment:
if this fails, a java exception has been set, right? Add a comment to that effect
Looking in the function, it does seem that the only things that can fail are the things that create java objects